### ISSUE TITLE
Segmentation fault: 11 when running onnx_to_coreml.py

### ISSUE THREAD

**Issue by AnkitPurohit01** [onnx/tutorials#76]
Hi,
I am trying ONNXLive Tutorial.
When I try to convert the ONNX models to CoreML models by running onnx_to_coreml.py script,
I get a segmentation 11 error.
I tried both python 2.7 and 3.6 but same error is occurring.

Following is the script I am trying to run : 

import sys
from onnx import onnx_pb
from onnx_coreml import convert

model_in = sys.argv[1]
model_out = sys.argv[2]

model_file = open(model_in, 'rb')
model_proto = onnx_pb.ModelProto()
model_proto.ParseFromString(model_file.read())
coreml_model = convert(model_proto, image_input_names=['0'], image_output_names=['186'])
coreml_model.save(model_out)

Please suggest me a solution.
Thank you so much in advance.

**Comment by houseroad** [comment]
@smessmer could you take a look?

**Comment by johnwayne1995** [comment]
I have the same issue.

**Comment by mnishiki** [comment]
I have the same issue.

**Comment by drogen120** [comment]
I have the same issue.

**Comment by Hugstar** [comment]
I've had the same issue. I was able to fix this by compiling ONNX from source.

**Comment by drogen120** [comment]
@Hugstar Thank you. I complied ONNX from the source but still the same error. I think the error comes from the convert function.

**Comment by Seanbakko** [comment]
Has anyone converted a pytorch model to CoreML successfully? Getting the same Segmentation fault: 11 error. Anyone know how to make this work? @drogen120 @Hugstar @AnkitPurohit01

**Comment by jaunruh** [comment]
Like mentioned in #365 from onnx/onnx-coreml, I am also still having the issue of a segfault. 
I was not able to convert any onnx model to a mlmodel.

**Comment by Hugstar** [comment]
@Seanbakko Yes - what fixed it for me was compiling **both** Pytorch and ONNX from source and it solved the issue completely.

**Comment by Umity** [comment]
> @Seanbakko Yes - what fixed it for me was compiling **both** Pytorch and ONNX from source and it solved the issue completely.

couldn't solve my problem. getting "Segmentation fault: 11" when trying to convert to coreml.. so annoying.
